datafusion/tests/parquet_pruning.rs

```diff
@@ -262,7 +262,7 @@ async fn prune_int32_scalar_fun() {
     println!("{}", output.description());
     // This should prune out groups with error, because there is not col to
     // prune the row groups.
-    assert_eq!(output.predicate_evaluation_errors(), Some(1));
+    assert_eq!(output.predicate_evaluation_errors(), Some(4));
```

Comment:
We are evaluating the filter for each row group now.  I think it's an expected change for evaluation errors.

Reply:
yeah, I agree.